api/src/upload/upload.py

```
@@ -0,0 +1,182 @@
+import rasterio
+import hashlib
+import shutil
+import time
+
+from rasterio.env import Env
+from rasterio.warp import transform_bounds
+
+from fastapi import HTTPException
+
+from shared.models import Dataset, StatusEnum
+from shared.supabase import use_client, login
```

Ruff (0.8.0)

`shared.supabase.login` imported but unused

Remove unused import: `shared.supabase.login`

(F401)